src/index.html

```diff
@@ -1,13 +1,64 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="content">
+      <div class="stars--0">
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+      </div>
+      <div class="stars--1">
+        <div class="light-star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+        <div class="star item"></div>
+      </div>
+      <div class="stars--2">
+        <div class="light-star item"></div>
```

Comment:
for each star should be just one class
for example stars__star.
for amount active stars should be responsible modifier class on parent element
```suggestion
        <div class="stars__star"></div>
```

Reply:
why if i need active or disable star? it just than use css selector